components/forms/iframe-form.jsx

```diff
@@ -1,4 +1,5 @@
 import React, { useEffect, useCallback, useRef } from 'react'
+import ResizeObserver from 'resize-observer-polyfill'
```

Comment:
Can this be implicitly imported through Webpack somehow?

Reply:
I followed the recommended way https://www.npmjs.com/package/resize-observer-polyfill#usage-example